README.md

```diff
@@ -396,6 +396,259 @@ public CloudTasksClient getService() throws IOException {
+Retry Delay Multiplier: 2.0
+Max Retry Delay: 500ms
+```
+
+- Attempt 1: Random value between [1, 100]
+- Attempt 2: Random value between [1, 200]
+- Attempt 3: Random value between [1, 400]
+- Attempt 3: Random value between [1, 500]
+- ...
+- Attempt X: Random value between [1, 500]
+
+### How to find the Retry Configurations for an RPC
+Default retry params are configured inside the client's generated StubSettings' class.
+
+#### For example, using Java-Asset v3.41.0:
+Retry Status Codes are configured [here](https://github.com/googleapis/google-cloud-java/blob/d9da511b4b56302e509abe8b2d919a15ea7dcae7/java-asset/google-cloud-asset/src/main/java/com/google/cloud/asset/v1/stub/AssetServiceStubSettings.java#L1058-L1082)
+
+Example:
+```java
+      definitions.put(
+          "retry_policy_1_codes",
+          ImmutableSet.copyOf(
+              Lists.<StatusCode.Code>newArrayList(
+                  StatusCode.Code.DEADLINE_EXCEEDED, StatusCode.Code.UNAVAILABLE)));
+```
+
+Retry Params are configured [here](https://github.com/googleapis/google-cloud-java/blob/d9da511b4b56302e509abe8b2d919a15ea7dcae7/java-asset/google-cloud-asset/src/main/java/com/google/cloud/asset/v1/stub/AssetServiceStubSettings.java#L1086-L1155)
+
+Example:
+```java
+      settings =
+          RetrySettings.newBuilder()
+              .setInitialRpcTimeout(Duration.ofMillis(60000L))
+              .setRpcTimeoutMultiplier(1.0)
+              .setMaxRpcTimeout(Duration.ofMillis(60000L))
+              .setTotalTimeout(Duration.ofMillis(60000L))
+              .build();
+```
+
+The default configurations are mapping [here](https://github.com/googleapis/google-cloud-java/blob/d9da511b4b56302e509abe8b2d919a15ea7dcae7/java-asset/google-cloud-asset/src/main/java/com/google/cloud/asset/v1/stub/AssetServiceStubSettings.java#L1306-L1474)
+
+Example:
+```java
+      builder
+          .exportAssetsSettings()
+          .setRetryableCodes(RETRYABLE_CODE_DEFINITIONS.get("no_retry_0_codes"))
+          .setRetrySettings(RETRY_PARAM_DEFINITIONS.get("no_retry_0_params"));
+```
+
+### Retry Examples
+The following examples below show the behavior of some retry configurations.
+
+Note: These examples below assume that jitter is disabled. The retry delay is computed to be the maximum value
+at the end of each attempt.
+
+#### No Retry
+```java
+RetrySettings defaultNoRetrySettings =
+        RetrySettings.newBuilder()
+        // ... Other configurations do not matter
+        .setTotalTimeout(Duration.ofMillis(5000L))
+        // Explicitly set retries as disabled (maxAttempts == 1)
+        .setMaxAttempts(1)
+        .build();
+```
+
+The following table shows the attempts:
+
+| Attempt Number 	| RPC Timeout 	| Retry Delay 	| Call Invoked 	| Call Ended 	|
+|----------------	|-------------	|-------------	|--------------	|------------	|
+| 1              	| 5000ms      	| 0ms         	| 0ms          	| 50000ms    	|
+
+#### Retry
+##### Example 1
+```java
+RetrySettings.newBuilder()
```

Comment:
Can we use the same `RetrySettings` for all examples? I think it would be easier to process.

Reply:
There are slight differences in each RetrySetting. I can try and highlight the differences to make it clearer.